templates/stats/publications.tpl

```diff
@@ -184,30 +184,33 @@
 					<pkp-table
 						labelled-by="publicationDetailTableLabel"
 						:class="tableClasses"
```

Comment:
Oki.. I guess the :attr.classes it was because of this one?

This seems to just change opacity on loading..  I don't think its necessary. It already shows loading clearly enough.

I would suggest just to remove that.